### ISSUE TITLE
Screen Reader: Many Pages: The visual headings are not announced as such

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77253]
## Action Performed:
1. Using TalkBack + Chrome, open the Expensify app
2. Note that the 'Travel and expense, at the speed of chat' visual heading is not accessible
3. Swipe to the link titles in the footer section
4. Listen to the TalkBack announcement

Other Occurrences:
1. On Onboarding: the title of each screen
2. On Inbox: the title of each screen
3. On Profile: the titles of the screen, 'Public' and 'Private'
4. On Settings: the titles 'Account and General'
5. On Status: the title of the screen
6. On Status - Clear After
7. On Search
8. On Preferences: the screen title and text 'App preferences'
9. On Security: Security options, Copilot: Delegated access
10. On Wallet: Bank accounts, Expensify Wallet (Beta)
11. On all screens with the main title
12. On Reports - Filter: To-do and Explore

## Expected Result:
The visual headings should be announced as headings, e.g.:
- Travel and expense, at the speed of chat
- titles of footer links

## Actual Result:
The visual headings are not announced as headings, e.g.:
- Travel and expense, at the speed of chat
- titles of footer links

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 1.3.1 Info and Relationships
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

## User impact:
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested resolution:
Ensure that all visual headings are announced as headings by using accessibilityRole="header".
https://reactnative.dev/docs/accessibility

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/ef66f4a1-0821-4add-a51f-be8ae495bfae)

https://github.com/user-attachments/assets/792f056f-b036-4dad-94fc-7194a6d29338

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7004796
**Issue reported by:** NA
**Slack conversation:** NA
